packages/cli-kit/src/public/node/ui.tsx

```diff
@@ -499,6 +499,7 @@ export function renderTable<T extends ScalarDict>({renderOptions, ...props}: Ren
 
 interface RenderTasksOptions {
   renderOptions?: RenderOptions
```

Comment:
I would prefer to add this to the renderOptions, Seems like it'd be a more widespread change